### ISSUE TITLE
Install with a different name

### ISSUE THREAD

**Issue by homocomputeris** [homebrew-ffmpeg/homebrew-ffmpeg#130]
## Is your feature request related to a problem? Please describe.

It would be great to have an option to install this formula with a different name and bins named differently. E.g., to avoid `Error: ffmpeg is already installed from homebrew-ffmpeg/ffmpeg!`

## Describe the solution you'd like

Have the _possibility_ to install formula as, e.g. `ffmpeg-opt` with binaries (symlinks) called `ffwhatever-opt`.
So, the system will simply have `ffmpeg` and `ffmpeg-opt` in parallel without causing issues.

## Describe alternatives you've considered

- Installing/reinstalling every time there is a conflict with the core packages.

**Comment by homocomputeris** [comment]
Can it be made another formula in the tap? This wouldn't break anything for the existing users.

As far as I understand, the static build does not include FDK AAC.

**Comment by retokromer** [comment]
> As far as I understand, the static build does not include FDK AAC.

No, because the license of libfdk_aac is not compatible with FFmpeg’s GPL.

**Comment by retokromer** [comment]
@slhck I guess, we could easily add a flag. When not set, the behaviour is as now; when set, then it uses another name. Should I prepare a PR?

**Comment by slhck** [comment]
@retokromer Yes, that would make sense! I am not passionate about a name for that flag.

**Comment by homocomputeris** [comment]
It would be great to get this feature.

**Comment by retokromer** [comment]
I would like to resolve the GitHub Actions CI issue before adding new features to the formula.
